src/styles/headerStyles.scss

```
@@ -0,0 +1,64 @@
+@import "./value.scss";
+
+.header {
+  font-weight: 500;
+  text-transform: uppercase;
+  font-size: 12px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: $White;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0
+    rgba(0, 0, 0, 0.05)
+  ;
+}
+
+.logo_img {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  display: inline-block;
+  text-align: center;
+}
+
+.nav__link {
```

use scss nesting
```suggestion
&__link {
```
inside .nav {}